x/wasm/keeper/keeper.go

```diff
@@ -122,21 +137,24 @@ func NewKeeper(
 	}
 
 	keeper := &Keeper{
-		storeKey:          storeKey,
-		cdc:               cdc,
-		wasmVM:            wasmer,
-		accountKeeper:     accountKeeper,
-		bank:              NewBankCoinTransferrer(bankKeeper),
-		portKeeper:        portKeeper,
-		capabilityKeeper:  capabilityKeeper,
-		messenger:         NewDefaultMessageHandler(router, channelKeeper, capabilityKeeper, bankKeeper, cdc, portSource, customEncoders),
-		queryGasLimit:     wasmConfig.SmartQueryGasLimit,
-		paramSpace:        paramSpace,
-		metrics:           NopMetrics(),
-		gasRegister:       NewDefaultWasmGasRegister(),
-		maxQueryStackSize: types.DefaultMaxQueryStackSize,
-	}
-	keeper.wasmVMQueryHandler = DefaultQueryPlugins(bankKeeper, stakingKeeper, distKeeper, channelKeeper, queryRouter, keeper).Merge(customPlugins)
+		storeKey:             storeKey,
+		cdc:                  cdc,
+		wasmVM:               wasmer,
+		accountKeeper:        accountKeeper,
+		bank:                 NewBankCoinTransferrer(bankKeeper),
+		accountPruner:        NewVestingCoinBurner(bankKeeper),
+		portKeeper:           portKeeper,
+		capabilityKeeper:     capabilityKeeper,
+		messenger:            NewDefaultMessageHandler(router, channelKeeper, capabilityKeeper, bankKeeper, cdc, portSource),
+		queryGasLimit:        wasmConfig.SmartQueryGasLimit,
+		paramSpace:           paramSpace,
+		metrics:              NopMetrics(),
+		gasRegister:          NewDefaultWasmGasRegister(),
+		maxQueryStackSize:    types.DefaultMaxQueryStackSize,
+		acceptedAccountTypes: defaultAcceptedAccountTypes,
+		pluskeeper:           pluskeeper,
```

Comment:
If there is not the `plusKeeper`, can this `x/wasm` module work? 
The default wasm's keeper does not need other plus keepers. But this changes breaks this structure of `x/wasm` module. Because in this case, this `x/wasm` module should add `plusKeeper`.